code/game/objects/items/weapons/storage/book.dm

```diff
@@ -105,7 +105,11 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
 		to_chat(user, "<span class='danger'>The book sizzles in your hands.</span>")
 		user.take_bodypart_damage(0,10)
 		return
-
+	
+	if (!heal_mode)
+		..()
+		return
```

Comment:
Can now confirm, removing the return breaks the item, causing a rather long delay that's different from how a regular bible works.